docker-compose.yml

```diff
@@ -54,6 +54,10 @@ services:
       - type: bind
         source: ./conf/etc/php/7.2/fpm/pool.d/www.conf
         target: /etc/php/7.2/fpm/pool.d/www.conf
+      # Directory for serving certbot challenge files
+      - type: bind
+        source: /var/www/challenges
```

Comment:
I think source and target are reversed..

Reply:
Yup fixed now.